src/components/table/table.scss

```diff
@@ -0,0 +1,58 @@
+:host {
+    display: block;
+  }
+  .pointer {
+    cursor: pointer !important;
+  }
+  .sort-desc {
+    transform: rotate(180deg);
+    /* background-color: yellow; */
+  }
+  
+  .sort-desc:hover {
+    cursor: pointer !important;
+    background-color: white;
+    color: #041e42;
+  }
+  
+  .sort-asc:hover {
+    cursor: pointer !important;
+    background-color: white;
+    color: #041e42;
```

Comment:
Some css that are added, for example color: #041e42, are css that should be added in scania-theme. **All scania** related css should be added in scania-theme repo https://github.com/scania/scania-theme . Only the default css should be left.

Reply:
Just make a fork from scania-theme and create a branch :)